pkg/expansion/aggregate.go

```diff
@@ -3,10 +3,15 @@ package expansion
 import (
 	"fmt"
 
+	"github.com/open-policy-agent/frameworks/constraint/pkg/instrumentation"
 	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
 )
 
-const childMsgPrefix = "[Implied by %s]"
+const (
+	childMsgPrefix = "[Implied by %s]"
+
+	ChildStatLabel = "EphemeralReviewFor"
```

Comment:
"ephemeral review" is what we were calling this during dev, but I'm not sure it means much to users.

"ImpliedBy" to keep the verbiage consistent? I'm open to better ideas.